### ISSUE TITLE
Guesses not checked against dictionary

### ISSUE THREAD

**Issue by davebrown-exit107** [burkee75/wordle-cli#6]
In the online version of wordle, you cannot guess non-dictionary words. It would be good to have the cli check to see if a word is in the dictionary before accepting it as a guess.

**Comment by burkee75** [comment]
Good catch. Looking at the built in dictionary (on MacOS) there are a lot of words that I don't think anyone would guess that I also want to trim out somehow.

**Comment by davebrown-exit107** [comment]
Might I recommend using [Fj00/CEL](https://github.com/Fj00/CEL) as a basis for your word list? I've used it for other projects and found it to be helpful. I'll also look back at some of my word game code to remember where I got some of my word lists from.
